src/worker.js

```diff
@@ -106,20 +129,28 @@ async function onCallbackQuery(callbackQuery) {
 
     console.log(`Check: ${title}, ${timeEstimate} ${orgName}:${repoName}`);
 
-    if (!repoName || !orgName) {
+    if (!repoName || !orgName)
+    {
       console.log(`No Github data mapped to channel`);
       return;
     }
 
-    const res = await createIssue(timeEstimate, orgName, repoName, title, replyToMessage, messageLink);
+    // get tagged user if available
+    const tagged = extractTag(replyToMessage);
+
+    // remove tag from issue body
+    const tagFreeTitle = removeTag(replyToMessage)
 
-    console.log(`Issue created: ${res.html_url}`);
+    const { data, assignees } = await createIssue(timeEstimate, orgName, repoName, title, tagFreeTitle, messageLink, tagged);
 
-    const msg = escapeMarkdown(`*Issue created: [Check it out here](${res.html_url})* with time estimate *${timeEstimate}*`, "*`[]()");
+    console.log(`Issue created: ${data.html_url}`);
+
+    const msg = escapeMarkdown(`*Issue created: [Check it out here](${data.html_url})* with time estimate *${timeEstimate}*${assignees ? ` and @${tagged} as assignee` : ''}`, "*`[]()");
```

Comment:
```
@${tagged} is assigned to [${title}]($githubUrl)

${timeEstimate}

[ View on GitHub ]
```

The view on GitHub button should be in the same format as the permit claim buttons. 

I would like to try a few messages with you to see how they render. 

Can you make sure that the link preview is not expanded on telegram? It makes the message look too big.

Reply:
> Can you make sure that the link preview is not expanded on telegram? It makes the message look too big.

Looks like there is no way to remove opengraph on links with the telegram api